build.sbt

```diff
@@ -2,12 +2,12 @@ import Tests._
 import scoverage.ScoverageKeys
 
 // All Twitter library releases are date versioned as YY.MM.patch
-val releaseVersion = "20.9.0-SNAPSHOT"
+val releaseVersion = "20.7.0"
 
 val libthriftVersion = "0.10.0"
 
-val defaultNetty4Version = "4.1.47.Final"
-val defaultNetty4StaticSslVersion = "2.0.30.Final"
+val defaultNetty4Version = "4.1.50.Final"
```

Comment:
Where is this getting used... 'defaultNetty4Version'?
That package name will appear in commit message, instead of 'Netty4Version'.